Sources/HtmlCssSupport/Support.swift

```diff
@@ -13,20 +13,25 @@ public func style<T>(_ style: Stylesheet) -> Attribute<T> {
   return .init("style", style)
 }
 
-public func style(_ attribs: [Attribute<Element.Style>], _ css: Stylesheet) -> ChildOf<Element.Head> {
-  return style(render(config: Css.compact, css: css))
+public func style(
+  _ attribs: [Attribute<Element.Style>],
+  _ css: Stylesheet,
+  config: Css.Config = .compact
```

Comment:
Exposing `config` publicly so we can render these in a pretty way for snapshots.